<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Consulta_Cnpj</class>
 <widget class="QFrame" name="Form_Consulta_Cnpj">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC INFORMÁTICA| CADASTRO CLIENTE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/clientes.png</normaloff>../../img/clientes.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="fr_FormClientes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <widget class="QLabel" name="lb_FormProdutos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>880</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

border-bottom: 2px solid #A2A2A2
}</string>
    </property>
    <property name="text">
     <string>CONSULTAR CNPJ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Id">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border: 1px solid #A2A2A2;
color: #000;
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Razao_Social">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>RAZÃO SOCIAL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Nome">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>NOME COMPLETO</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Cnpj">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>CNPJ</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Edereco">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>630</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>ENDEREÇO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Cep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string>99999-999</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>123456789</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Cep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>CEP</string>
    </property>
   </widget>
   <widget class="QFrame" name="fr_BotoesFormProdutos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1000</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#E1DFE0;
border: none;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Tx_Cnpj">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string>00.000.000/0000-00</string>
    </property>
    <property name="text">
     <string>../-</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Telefone">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>TELEFONE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Telefone">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>196</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string>(00) 0000-00000</string>
    </property>
    <property name="text">
     <string>() -</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="cursorPosition">
     <number>15</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Logradouro">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>LOGRADOURO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txLogradouro">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Numero">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>Nº</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Numero">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Bairro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>260</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Bairro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>BAIRRO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_municipio">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>260</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Municipio">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>MUNICIPIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Uf">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>UF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Uf">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_BuscaCep_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Salvar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Voltar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Consultar_Cnpj">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>88</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Pesquisar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:none;
 }
QPushButton:hover{
background-color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/pesquisar.png</normaloff>../../img/pesquisar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_Id</tabstop>
  <tabstop>Tx_Cnpj</tabstop>
  <tabstop>tx_Nome</tabstop>
  <tabstop>tx_Email</tabstop>
  <tabstop>tx_Telefone</tabstop>
  <tabstop>tx_Cep</tabstop>
  <tabstop>txLogradouro</tabstop>
  <tabstop>tx_Numero</tabstop>
  <tabstop>tx_Bairro</tabstop>
  <tabstop>tx_municipio</tabstop>
  <tabstop>tx_Uf</tabstop>
  <tabstop>bt_BuscaCep_2</tabstop>
 </tabstops>
 <resources>
  <include location="../Icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
